main.go

```diff
@@ -276,41 +281,66 @@ func downloadSourcePaths(sourcePaths []string, destPath string, githubRepo GitHu
 
 // Download the specified binary file that was uploaded as a release asset to the specified GitHub release.
 // Returns the path where the release asset was downloaded.
-func downloadReleaseAsset(assetName string, destPath string, githubRepo GitHubRepo, tag string) (string, error) {
-	var assetPath string
+func downloadReleaseAssets(assetRegex string, destPath string, githubRepo GitHubRepo, tag string) ([]string, error) {
+	var err error
+	var assetPaths []string
 
-	if assetName == "" {
-		return assetPath, nil
+	if assetRegex == "" {
+		return assetPaths, nil
 	}
 
-	release, err := GetGitHubReleaseInfo(githubRepo, tag)
-	if err != nil {
-		return assetPath, err
+	release, releaseInfoErr := GetGitHubReleaseInfo(githubRepo, tag)
+	if releaseInfoErr != nil {
+		return assetPaths, err
 	}
 
-	asset := findAssetInRelease(assetName, release)
-	if asset == nil {
-		return assetPath, fmt.Errorf("Could not find asset %s in release %s", assetName, tag)
+	assets, err := findAssetsInRelease(assetRegex, release)
+	if err != nil || assets == nil {
+		return assetPaths, fmt.Errorf("Could not find assets matching %s in release %s", assetRegex, tag)
 	}
 
-	assetPath = path.Join(destPath, asset.Name)
-	fmt.Printf("Downloading release asset %s to %s\n", asset.Name, assetPath)
-	if err := DownloadReleaseAsset(githubRepo, asset.Id, assetPath); err != nil {
-		return assetPath, err
+	var wg sync.WaitGroup
+	var errorStrs []string
+
+	for _, asset := range assets {
+		wg.Add(1)
+		go func(asset *GitHubReleaseAsset) {
+			assetPath := path.Join(destPath, asset.Name)
+			fmt.Printf("Downloading release asset %s to %s\n", asset.Name, assetPath)
+			if downloadErr := DownloadReleaseAsset(githubRepo, asset.Id, assetPath); downloadErr == nil {
+				assetPaths = append(assetPaths, assetPath)
```

Comment:
IIRC, `append` is not goroutine safe. Can you either use a mutex to guard this section, or use channels to collect the results?